tests/integration/components/slots-picker/desktop-no-category-column/component-test.js

```diff
@@ -0,0 +1,103 @@
+import {
+  module} from 'qunit'; import {setupRenderingTest
+} from 'ember-qunit';
+import { test } from 'qunit';
+import hbs from 'htmlbars-inline-precompile';
+import { generateAppointmentSlots } from 'ember-appointment-slots-pickers/test-support/helpers/generate-appointment-slots';
+import { run } from '@ember/runloop';
+import { render, settled, findAll } from '@ember/test-helpers';
+
+module('Integration | Component | slots-picker/desktop-no-category-column', function (hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function () {
+    this.actions = {};
+    this.send = (actionName, ...args) => this.actions[actionName].apply(this, args);
+  });
+
+  test('with slots available, no slot selected', async function (assert) {
+
+    generateAppointmentSlots.call(this, {
+      numberOfAppointments: 50
+    });
+    this.set('testSelect', () => {});
+
+    await render(hbs`
+      <div>
+        {{#slots-picker
+          appointmentSlots=generatedAppointmentSlots
+          select=(action testSelect)
+          as |baseProps onSelectSlot|
+        }}
+          {{slots-picker/desktop-no-category-column
```

Comment:
try to add this version to the dummy app demo? Not a blocker

Reply:
@Leooo  i have added it already

Reply:
ah true in https://github.com/britishgas-engineering/ember-appointment-slots-pickers/pull/37/files#diff-c04908ab51c6d117a409183f4bc67331R10 thanks